test/integration/volumes_test.go

```diff
@@ -130,6 +130,30 @@ func TestWaitForVolumeLinodeIDInstance(t *testing.T) {
 	}
 }
 
+func TestUpdateVolume(t *testing.T) {
+	client, teardown := createTestClient(t, "fixtures/TestUpdateVolume")
+	defer teardown()
+
+	updateOpts := linodego.VolumeUpdateOptions{
+		Label: "my-testing-volume",
+	}
+	volume, err := client.UpdateVolume(context.Background(), 123456, updateOpts)
+	if err != nil {
+		t.Errorf("Error updating volume, expected struct, got error %v", err)
+	}
+	if volume.ID == 0 {
+		t.Errorf("Expected a volumes id, but got 0")
+	}
+	if volume.Label != "my-testing-volume" {
+		t.Errorf("Expected volume label to be equal to updated label")
+	}
+	assertDateSet(t, volume.Created)
+	assertDateSet(t, volume.Updated)
+	if err := client.DeleteVolume(context.Background(), volume.ID); err != nil {
```

Comment:
If you use `setupVolume`, the teardown will handle cleaning up the volume for us so this block won't be needed

Reply:
definitely cleans things up :) :+1: